unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/UserContextMenu/Scripts/UserContextMenu.cs

```diff
@@ -93,6 +94,18 @@ public enum MenuConfigFlags
     private bool isFriendsEnabled => DataStore.i.featureFlags.flags.Get().IsFeatureEnabled("friends_enabled");
     internal ISocialAnalytics socialAnalytics;
 
+    private IFriendsController _friendsController;
+
+    private IFriendsController friendsController
```

Comment:
We should try to inject this reference instead of using the static accessor.

Reply:
This is out of scope of the PR